README.md

```diff
@@ -2,43 +2,44 @@
 
 This module provides the required Azure resources to integrate an Azure subscription with Uptycs.
 
+## Prerequisites [Do Not Skip]
 
-It creates the following resources:
+Ensure you have the following privileges before you execute the Terraform Script:
 
-* Application
-* Service principal to the application
-* It attaches the following roles and permissions to the service principal:
+- Administrative roles:
+  - `Privileged Role Administrator` (AD Role)
+  - `Application Administrator` (AD Role)
+- `Owner` role at the subscription scope which is being integrated
 
-  **Roles**:
+Absence of the above privileges will result in Access related issues when trying to run the Terraform.
```

Comment:
Absence of the above privileges may result in access related issues when trying to run the Terraform.

Reply:
got it